google/cloud/internal/credentials_impl.cc

```diff
@@ -84,6 +84,9 @@ ServiceAccountConfig::ServiceAccountConfig(std::string json_object,
     : json_object_(std::move(json_object)),
       options_(PopulateAuthOptions(std::move(opts))) {}
 
+ExternalAccountConfig::ExternalAccountConfig(std::string j, Options o)
+    : json_object_(std::move(j)), options_(PopulateAuthOptions(std::move(o))) {}
```

Comment:
nit: I applaud you for saving a line, but `j -> json_object`, `o -> opts` would be more informative and more consistent with the method signature for `ServiceAccountConfig`.

Reply:
Fixed